proposal/proposal.Rmd

```diff
@@ -10,7 +10,7 @@ At a high-level address what your proposal is and how it will address the proble
 
 This proposal is for funding to support the development of coding skills for increasing the participation of women in the R community. There are two levels of action:
 
-1. Building the package development skills of career-young women in the research community, to encourage them to distribute their research as an R package.
+1. Building the package development skills of women in the research community, to encourage them to distribute their research as an R package.
```

Comment:
I didn't achieve what we're trying to foster here until my 40s, post-tenure. I'd rather not make it explicitly about *young* women, though I am sure these workshops will draw a fairly young crowd, which is great!